approved/k8s-delete.yaml

```diff
@@ -0,0 +1,27 @@
+# kubernetes delete / cleanup - release name
+    - action:
+        uses: kubernetes-delete
+        with:
+          release: release-name
+          namespace: true
```

Comment:
should this be named `deleteNamespace` ? since we could also have the infra namespace in the YAML (in case the user wants to).

Reply:
@abhinavsingh1196 we need it to be kebob case. Also the context is delete so we are saying we are deleting the namespace. We can be explicit and say `delete-namespace` if you want.